### PR TITLE
Added support for coveralls.io

### PR DESCRIPTION
**Pull request by jhellar**
# Motivation
https://issues.jboss.org/browse/FH-2619
Add code coverage status with coveralls.io.

# Change
Modified travis config to generate also code coverage and send that to coveralls.io.
Added badge to README.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,11 +4,17 @@ xcode_workspace: FeedHenry.xcworkspace
 xcode_scheme: FeedHenry
 xcode_sdk: iphonesimulator
 
+before_install:
+  - brew install macmade/tap/xcode-coveralls
+
+script:
+  - xctool -workspace FeedHenry.xcworkspace -scheme FeedHenry build test -sdk iphonesimulator GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES
```
Comment (corinnekrych) [inline .travis.yml]: @jhellar With Xcode 8 and Swift3 update coming very soon, xctool is not supported anymore (see: https://github.com/facebook/xctool/issues/704) could you use xcodebuild instead pls

**Comment by corinnekrych** [comment]
@jhellar Looking at the detailled coverage report https://coveralls.io/builds/8406958
I don't see any of the Swift files. Only the Objective-C has been instrumented.
for the fh-ios-swift-sdk only the sync code has been left in Objective-C, all the other part, ie: cloud call, oauth, push are in swift.

**Comment by jhellar** [comment]
@corinnekrych should be fixed. I managed to get this work with slather.

**Comment by corinnekrych** [comment]
@jhellar why do we have files like:
/../../../../../Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/usr/include/sys/_types.h
included in the coverage report?

**Comment by jhellar** [comment]
@corinnekrych I did not notice and I'm not sure, thought that these should be ignored because of "source_directory: FeedHenry" line in .slather.yml. I will look at this.

**Comment by jhellar** [comment]
@corinnekrych should be fixed now.

**Comment by corinnekrych** [comment]
+1
thanks!!! @jhellar 
🌹 🌹

**Pull request merged**